ipam/client/tests/test_phpipam.py

```diff
@@ -143,6 +143,115 @@ def test_add_next_ip(testphpipam):
         testphpipam.add_next_ip(subnet, 'err', 'err')
     assert "is full" in str(excinfo.value)
 
+    subnet = IPNetwork('2001::40/125')
```

Comment:
**OPT**: use `2001:db8::/32` (the block reserved for doc & tests as per RFC-3849)

Reply:
Thanks for the pointer. Will do in another PR later on.